helm/nginx-ingress-controller-app/templates/controller-deployment.yaml

```diff
@@ -128,13 +123,42 @@ spec:
           {{- end }}
         {{- end }}
         {{- end }}
+        {{- if .Values.controller.livenessProbe.enabled }}
```

Comment:
Supported disabling every probe for consistency. Most importantly it's introduced for disabling liveness (and optionally and enabling only startup probe) on demand for a given tenant cluster.
This is to give us extra option to try if liveness probe resilience improvements prove not to be enough.